test/make_and_run_bmi_unit_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+gcc -lm main_unit_test_bmi.c ../src/bmi_cfe.c ../src/cfe.c -o run_cfe_bmi_test
+./run_cfe_bmi_test ./configs/cat_89_bmi_config_cfe.txt
```

Comment:
In continuation of the comment above, we should be able to run pointing to a config file in the other configs directory. We can even have a config file for the unit test: 
`./run_cfe_bmi_test ../configs/cat_89_bmi_config_cfe_unit_test.txt`  

I don't know what would be better, just bringing it up for discussion.

Reply:
will update.  good suggestion.